### ISSUE TITLE
[mac1] Can't log on to LINE.me desktop app

### ISSUE THREAD

**Issue by 130s** [kinu-garage/hut_10sqft#505]
I confirmed I can log on to my line.me account from my Android using FB account.

On their OSX app, which I just installed, and successfully reset the password on their web site, but it still doesn't let me logon saying "Email address or password is either incorrect or not registered with LINE".

Did all the support page suggests https://help.line.me/line/mac/pc?lang=en but no avail yet.
> - Uninstall LINE for Windows or Mac and then reinstall it.
> - Restart your computer. 
> - Temporarily disable any firewalls or security software you have in place, or allow LINE to connect to the internet.
> - Disable any resident or customization software such as WindowsBlinds.

**Comment by 130s** [comment]
Figured out somehow.

On Android app, I registered email to my account and set password, using the same email I just rest the password for from OSX. Then, on OSX app, using the same email and password, the app let me log in this time.

Strange. To log on to the Android app, I used my FB account, which uses the same email. So I assumed for the "account" on LINE on Android the same email is already registered, but looks like that's not the case. Like mentioned above, I had to manually register the email.
